### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by luxuriant777**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command):
+    if command.split()[0] == "mv":
+        file = command.split()[1]
+        path = command.split()[2]
```
Comment (TPolina) [inline app/main.py:7]: you use `split` 3 times while you can use it once

File: app/main.py [hunk 2]
```diff
@@ -1 +1,17 @@
...
+        new_file = path.split("/")[-1]
+        path = path.replace(new_file, "")
+        new_path = os.path.join(path)
+        os.makedirs(new_path)
+        with open(file, "r") as source_file:
+            file_copy = source_file.read()
+        os.remove(file)
+        os.chdir(new_path)
+        with open(new_file, "w") as new_file:
+            new_file.write(file_copy)
```
Comment (TPolina) [inline app/main.py:17]: too much code in if block but it's not necessary here. Can you refactor it?

**Review by TPolina — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (luxuriant777) [reply app/main.py]: changed

**Review by TPolina — APPROVED** [review]
Good job!

**Pull request closed without merging**